generators/entity-server/files.js

```diff
@@ -0,0 +1,240 @@
+/**
+ * Copyright 2013-2018 the original author or authors from the JHipster project.
+ *
+ * This file is part of the JHipster project, see http://www.jhipster.tech/
+ * for more information.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+const _ = require('lodash');
+const randexp = require('randexp');
+const chalk = require('chalk');
+const fs = require('fs');
+const utils = require('generator-jhipster/generators/utils');
+const constants = require('generator-jhipster/generators/generator-constants');
+
+/* Constants use throughout */
+const INTERPOLATE_REGEX = constants.INTERPOLATE_REGEX;
+const SERVER_MAIN_RES_DIR = constants.SERVER_MAIN_RES_DIR;
+const TEST_DIR = constants.TEST_DIR;
+const SERVER_MAIN_SRC_KOTLIN_DIR = `${constants.MAIN_DIR}kotlin/`;
+const SERVER_TEST_SRC_KOTLIN_DIR = `${constants.TEST_DIR}kotlin/`;
+
+// const BASE_DIR = '../../../node_modules/generator-jhipster/generators/entity-server/templates/';
```

Comment:
change this into > `const BASE_DIR = '../../../../../../node_modules/generator-jhipster/generators/entity-server/templates';`